packages/workbox-broadcast-cache-update/broadcastUpdate.mjs

```diff
@@ -55,6 +56,17 @@ import './_version.mjs';
  * @memberof workbox.broadcastUpdate
  */
 const broadcastUpdate = (channel, cacheName, url, source) => {
+  // There are browsers which support service workers but don't support the
+  // Broadcast Channel API.
+  // See https://github.com/GoogleChrome/workbox/issues/1304
+  if (!(('BroadcastChannel' in self) && channel)) {
```

Comment:
This just feels error prone - why don't we just check for no channel?

The current AND'ing of support with channel just feels OTT. We know our code won't have channel when the `BroadcastChannel` API is unavailable.